mono/mini/aot-runtime.c

```diff
@@ -3313,9 +3315,11 @@ mono_aot_find_jit_info (MonoDomain *domain, MonoImage *image, gpointer addr)
 
 	nmethods = amodule->info.nmethods;
 
+#ifndef ENABLE_INTERPRETER
```

Comment:
The wrappers will only work in the root domain, so the loading code needs to be modified somewhere to look for them in the root domain. i.e. see the 'target_domain' variable
in mono_jit_compile_method_with_opt ().